vpc/allocate/allocate_network.go

```diff
@@ -51,10 +51,15 @@ var AllocateNetwork = cli.Command{ // nolint: golint
 			Usage: "How long to wait for security groups to converge, in seconds",
 			Value: 10 * time.Second,
 		},
+		cli.DurationFlag{
+			Name:  "wait-for-sg-lock-timeout",
+			Usage: "How long to wait for other users, if the SG is in use",
```

Comment:
In what scenario would we now expect contention on the same SG? If the SG should be released prior to signaling master of completion, is this just waiting in unexpected/error scenarios?

Reply:
This timeout will never get hit in the case of the same set of SGs. Only when changing SGs. It's basically, because v2 engine is going to stick around for the forseeable, and we need a solution for that. This is kind of like launchguard-lite.